movie/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { Container } from "@material-ui/core";
+import React from "react";
+import styled from "styled-components";
+import Logo from "../header/Logo/Logo";
+
+const Footer: React.FC = () => {
+  return (
+    <FooterSection>
+      <Container maxWidth="sm">
```

Comment:
You should use for example <StyledContainer maxWidth="sm"> instead of nested wrapper.
Look at this docs:
https://styled-components.com/docs/advanced#caveat

Reply:
changed